tracepath.c

```diff
@@ -284,8 +284,9 @@ static int recverr(struct run_state *const ctl)
 		struct timespec res;
 
 		timespecsub(&ts, retts, &res);
-		printf(_("%3ld.%03ldms "), res.tv_sec * 1000 + res.tv_nsec / 1000000,
-					   (res.tv_nsec % 1000000) / 1000);
+		printf(_("%3lld.%03ldms "), (long long int)res.tv_sec * 1000
```

Comment:
@nmeyerhans @okias Any opinion if we need to translate this? Does `ms` (as millisecond) really need to be translated? Some languages (ka, ja) translate it. We don't translate `"%ld %d %d\n"` in `clockdiff.c`

Reply:
My inclination is that we don't need to translate this. I expect that "ms" is widely recognized and understood.  However, please note that I'm a native english speaker and don't consider myself much of an authority when it comes to localization matters.  The "C" locale is sufficient for me. :)

Reply:
@yurchor @rffontenelle @marek22k @Jibec @simmon-nplob @oersen @NorwayFun @robert-scheck any oppinion about this, please?

Similar would be `"host=%s rtt=%ld(%ld)ms/%ldms delta=%dms/%dms %s\n"`

Reply:
In the German language, we use "ms". Not sure if I would translate "host", "rtt" and "delta" into German, because "host" would likely end up with "server" (well, replacing one English technical word with another), "rtt" has no German abbreviation (I'm personally even not used to the German long form) and "delta" is at least in German known from the Greeks related to mathematics and science.

However, from my understanding, "ms" might be translated in languages that don't use latin characters (e.g. Macedonian uses "мс" according to Wikipedia), but I'm not experienced with this.

Reply:
The "мс" is ms in Cyrilic script, so all countries that use Cyrilic use that.

Also ping on windows seems to be fully localized at least in Czech https://idoc.vsb.cz/xwiki/bin/view/tuonet/sit-nastaveni/ping/?language=cs not sure if doing that for iputils is worth the effort. At least I can see all the bug reports that we broke shell script that are parsing the output from ping/tracepath/etc.

Reply:
No objections from me.

Reply:
LGTM.

Reply:
LGTM

Reply:
Did I understand properly that we agree _not_ to translate `ms`? Because it is important to have common output. Also, some translations just add space between number and `ms`. I can create a ticket for solution we agree on I did not understood whether we agree on.

I wonder if I force myself to do ping output in json. It would not be translated and it would be a proper solution for scripts.